controller/src/beerocks/master/db/db.cpp

```diff
@@ -2516,6 +2543,16 @@ bool db::set_client_time_life_delay(const sMacAddr &mac,
     if (save_to_persistent_db) {
         LOG(DEBUG) << __FUNCTION__
                    << ", configuring persistent-db, timelife = " << time_life_delay_sec.count();
+
+        std::unordered_map<std::string, std::string> values_map;
+        values_map.at("timestamp") = timestamp_to_string_seconds(timestamp);
```

Comment:
Why do you need a function to convert the timestamp to seconds?

Reply:
I am using this conversion in many places in the code so I thought it would be beneficial to have:
1. meaningful function to understand the operation performed
2. single-point-of-change in case we wish to change how we save the timestamp in the future